### ISSUE TITLE
Add support for includes

### ISSUE THREAD

**Issue by delx** [axkibe/lsyncd#102]
It's possible to exclude certain directories, but it's not possible to include certain directories.

I have the following problem: I'd like to sync /var/www from one server to another. I'd like to sync the /var/www/main folder, but not all other folders. I can't use exclude, because folders are created from time to time in /var/www and I'd have to add all new folders to the exclude list. Thus I need an include option! :)

**Comment by axkibe** [comment]
So 6 years later, this is now possible with git head.

the option is called filter and used like this:
```
sync {
	default.rsync,
	delay = 1,
	source = "/home/axel/lsyncd/src",
	target = "/home/axel/lsyncd/trg",
	filter = { '+ /*/', '+ /*/.ssh/', '+ /*/.shh/authorized_keys', '- **' }
}
```

Why did it take me 6 years? Because it was now that I needed this feature for my job :-)

**Comment by axkibe** [comment]
Just released 2.2.3 which is now able to do it.

In case of filters, be sure to also include all subpaths of a path. See example of my currently running usecase above.
